### ISSUE TITLE
Advanced search fields collapse together

### ISSUE THREAD

**Issue by czhci** [NEUBIAS/bise#172]
on the “Advanced search” page, when multiple options selected, the field box extends and covers the lines below, see picture below:

see [here](http://biii.eu/all-content?type=All&field_has_function_target_id%5B%5D=4111&field_has_function_target_id%5B%5D=4010&field_has_function_target_id%5B%5D=4057&field_has_function_target_id%5B%5D=4756&field_type_target_id=All).

**Comment by miura** [comment]
@czhci  maybe this is a duplicate of #163 ?

**Comment by czhci** [comment]
let's close this for now, and check after #163 is fixed.
